app/jobs/notify_web_hook_job.rb

```diff
@@ -42,6 +48,47 @@ def authorization
     Digest::SHA2.hexdigest([rubygem.name, version.number, webhook.api_key].compact.join)
   end
 
+  def hook_relay_url
+    "https://api.hookrelay.dev/hooks/#{ENV['HOOK_RELAY_ACCOUNT_ID']}/#{ENV['HOOK_RELAY_HOOK_ID']}/webhook_id-#{webhook.id}"
+  end
+
+  def use_hook_relay?
+    # can't use hook relay for `perform_now` (aka an unenqueued job)
+    # because then we won't actually hit the webhook URL synchronously
```

Comment:
Is that actual problem? Bypassing hookrelay makes sense in development environment for simple testing, but is there any reason to bypass that in production? Looking at the code, testing webhook (calling `/api/v1/web_hooks/fire`) is the only place using `perform_now`. Would it make any difference to call hookrelay in there as well? Any idea what's the delay?

Reply:
We use the return value from performing the hook in the response to the `fire` endpoint

Reply:
OK, let's leave it as is than. Would be great to find out way to remove this later.